File: Doeunnkimm/1주차/p_주식가격.js
function solution(prices) {
  let ans = []

  for (let i = 0; i < prices.length; i++) {
    let count = 0
    for (let j = 0; j < prices.length - i - 1; j++) {
      count++
      if (prices[i] > prices[i + j + 1]) {
        break
      }
    }
    ans.push(count)
  }
  return ans
}

File: seungri2/11주차셋째주/p_주식가격.py
def solution(prices):
    answer = [0]*len(prices)
    s = []
    
    for i, p in enumerate(prices):
        while s and prices[s[-1]] > p:
            j = s.pop()
            answer[j] = i-j            
        s.append(i)
    
    while s:
        i = s.pop()
        answer[i] = len(prices)-1-i
    
    return answer